features/ubuntu_pro_fips.feature

```diff
@@ -0,0 +1,505 @@
+Feature: Command behaviour when auto-attached in an ubuntu PRO fips image
+
+    @series.lts
+    @uses.config.machine_type.aws.pro.fips
+    Scenario Outline: Proxy auto-attach in an Ubuntu pro AWS machine
```

Comment:
I don't think we need the proxy tests for the fips pro machines. What do you think @orndorffgrant @renanrodrigo ?